docs/v0.4/getting-started/docker-for-windows.md

```diff
@@ -36,53 +27,67 @@ Enable Kubernetes in the Kubernetes tab, click on Apply, and wait for the instal
 
 ![enable Kubernetes](/img/docker-kubernetes-windows.png)
 
-If you previously had minikube or GKE configured, switch your kubectl context to "docker-desktop" using a PowerShell or command window.
+Confirm that your kubectl context is pointing to "docker-desktop".
+
+```sh
+kubectl config current-context
+```
+
+If you previously had a different cluster configured, switch your kubectl context to "docker-desktop" using a PowerShell or command window.
 
 ```sh
 kubectl config use-context docker-desktop
 ```
 
 ### monitor your cluster
-At this point it is useful to monitor your Kubernetes cluster. This PowerShell function will call `kubectl get` every 3 seconds.
+At this point it is useful to monitor your Kubernetes cluster. If you have [git bash](https://gitforwindows.org/) installed, create a bash script called `watch`, with the following content.
 
-```powershell
-function watchpods { while(1){ kubectl get pod --all-namespaces; start-sleep -seconds 3; clear }}
+```sh
```

Comment:
are we intentionally dropping the powershell equivalent?

Reply:
yes - see above - i don't like the flashing and the cost of installing git bash seems low enough to justify the better ux - but if you disagree we can revert to the old one and (someday) fix it to flash less.